.github/workflows/solidity.yml

```diff
@@ -9,6 +9,18 @@ defaults:
     shell: bash
 
 jobs:
+  readonly_changes:
+    name: Detect readonly solidity file changes
+    runs-on: ubuntu-latest
+    outputs:
+      changes: ${{ steps.ch.outputs.changes }}
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
+      - name: Detect readonly solidity file changes
+        id: ch
+        uses: ./.github/actions/detect-solidity-readonly-file-changes
+
   changes:
```

Comment:
it does seem there is quite a bit of dependency from other jobs from this? is there any implication from removing the `read-only` contract files from the check?

@erikburt @snehaagni do you guys have any idea?

Reply:
I believe this changes check is only for solidity.yml. Do you see it being used anywhere else?